syft/messaging/message.py

```diff
@@ -1052,7 +1056,9 @@ def detail(worker: AbstractWorker, msg_tuple: tuple) -> "CryptenInitPlan":
         Examples:
             message = detail(sy.local_worker, msg_tuple)
         """
-        return CryptenInitPlan(sy.serde.msgpack.serde._detail(worker, msg_tuple[0]))
+        msg_tuple = sy.serde.msgpack.serde._detail(worker, msg_tuple[0])
+        *context, model = msg_tuple
+        return CryptenInitPlan(tuple(context), model)
```

Comment:
You don't need to add an extra `tuple(...)`  context is already a tuple

Reply:
It is a list.
```msg_tuple can be = ({0: 'alice', 1: 'bob'}, 2, '127.0.0.1', 8080, b'test binary model')``` (taken from tests)
and after doing
```*context, model = msg_tuple```

```context```  would become ```[{0: 'alice', 1: 'bob'}, 2, '127.0.0.1', 8080]```